docker/compose.testing.yaml

```
@@ -0,0 +1,121 @@
+      - minio_test
+      - postgres_test
+      - redis_test
+    # https://docs.docker.com/reference/compose-file/services/#restart
+    restart: no
+  # https://github.com/minio/minio/issues/13025
+  minio-test:
+    # https://docs.docker.com/compose/environment-variables/
+    # https://docs.docker.com/reference/compose-file/services/#environment
+    environment:
+      - MINIO_BROWSER=${MINIO_BROWSER:?error}
+        # https://min.io/docs/minio/linux/reference/minio-server/settings/root-credentials.html#envvar.MINIO_ROOT_PASSWORD
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)